src/megachatapi.h

```diff
@@ -157,6 +204,64 @@ class MegaChatSession
      * @return true if audio is detected for this session, false in other case
      */
     virtual bool getAudioDetected() const;
+
+    /**
+     * @brief Returns a bit field with the changes of the session
+     *
+     * This value is only useful for session notified by MegaChatCallListener::onChatSessionUpdate
+     * that can notify about session modifications. The value only will be valid inside
```

Comment:
This value is only useful for session notified by MegaChatCallListener::onChatSessionUpdate ~that can notify about session modifications~.